docs/content-services/content-action.component.md

```diff
@@ -152,14 +153,63 @@ type and other details of the item just deleted:
 
 ![Custom delete message screenshot](../docassets/images/ContentActSnackMessage.png)
 
-### Built-in action examples
+### Examples
+
+#### System handler
+   
+This action simply execute one of the f built-in actions described above:
```

Comment:
"...one of the f built-in actions..." <- I guess the "f" should be removed or it is a typo (should be a digit, maybe?)